smart_proxy_dynflow_core.gemspec

```diff
@@ -22,20 +22,5 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = '~> 2.5'
 
   gem.add_development_dependency "bundler", ">= 1.7"
```

Comment:
Not sure if we even need this

Reply:
now we don't given we don't allow this deployed separatelly, `smart-proxy` should take care of that.